docs/CLI.md

```diff
@@ -53,6 +53,8 @@ To skip all (y/n) prompts, the `-y` keyword can be given:
 evergreen patch -y
 ```
 
+The patch command ignores warnings. If you want to view warnings, look at the [validate](#validating-changes-to-config-files) command.
```

Comment:
This assumes that the user already is aware that `evergreen patch` will output project validation errors, which I assume most people aren't aware of. The placement is also a little weird because it comes after the "basic usage" tutorial. I would move this to the beginning of the "basic usage" section and add a brief sentence prefacing what the patch command does to start off. For example, something like:

> `evergreen patch` allows you to submit patches to test your local changes. It will also check your project YAML any for validation errors before submission.

Reply:
@Kimchelly should I reference the validate command at all?

> evergreen patch allows you to submit patches to test your local changes. It will also check your project YAML any for validation errors before submission. If you want to view warnings, look at the [validate]() command.

Or should I keep it succinct like yours?

Reply:
Hmm, I think it still makes sense to reference `evergreen validate` since people may want a more complete validation rather than just patch-blocking errors.